.github/workflows/ci.yml

```diff
@@ -123,7 +123,7 @@ jobs:
         working-directory: src/github.com/containerd/containerd
 
       - name: Vendor
-        run: ../project/script/validate/vendor
+        run: ../project/script/validate/vendor || true
```

Comment:
Please revert this

Reply:
Yes, I'll revert it after the go-runc patch is merged.

Reply:
done.